sale_expected_delivery_date/README.rst

```diff
@@ -0,0 +1,47 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Sales Expected Delivery Date
+============================
+
+This module provides "Expected Delivery Date" in Sales Order.
+This date will be used for DO's Scheduled Time instead of standard
+use of Product's customer lead time (will be ignored).
```

Comment:
Please explain better how is done the calculation. Add section "Usage" a explain what happens with other dates in Sale Order: requiested date, commitment date, effective date.

![2016-03-07_7-46-42](https://cloud.githubusercontent.com/assets/8736623/13562443/f99fe28e-e438-11e5-9634-d5c1e29b7987.png)

This date is also used in Manufacturing, changing this field will impact not only in DO (Delivery Orders) so you must inform in README to be careful after installing this module.